pkg/controllers/ticdc/tasks/pod.go

```diff
@@ -37,6 +39,8 @@ import (
 
 const (
 	metricsPath = "/metrics"
+
+	defaultGracefulShutdownSeconds = 30
```

Comment:
Do we need to use the default timeout (10m) as v1?

Reply:
When I try to delete the whole cluster, deletion of ticdc will be blocked and the prestop hook cannot pass.

Reply:
I'll fix it in another PR